internal/planner/planner.go

```diff
@@ -322,14 +332,10 @@ func (p *Planner) planRules(rules []*ast.Rule) (string, error) {
 		}
 	}
 
+	p.loc = ploc // location of the ReturnLocalStmt is its call site -- TODO(sr): good choice?
```

Comment:
Nit: I'd prefer if the location was associated with the rule (either the head or the last line in the body). Otherwise we'll get different location info depending on the order we plan the rules in (I think--because if you have multiple calls to the same rule we only plan the rule once...)

Reply:
Good point.

> either the head or the last line in the body

We'll now record the first rule's location, and use that. (I think it makes more sense than the last rule, or default rule if there is one.) WDYT?

Reply:
The first rule's location makes the most sense. I hadn't thought it through completely but I now realize the last line wouldn't make sense because this will apply to all types of rules.